seedlink_plotter/seedlink_plotter.py

```diff
@@ -222,14 +232,24 @@ def plot_drum(self, stream):
             title += ' - autoscale -'
         title += " without filtering"
         self.figure.clear()
+
+        for tr in stream:
+            tr.stats.processing = []
+        
+        # adjust scale
+        if self.args.relative_scale:
+            scale = max(stream.std())*self.args.scale
```

Comment:
Please adhere to PEP8 for all new code. Here that means one space around `*` operator. Check out `flake8` or `black` for conda. Also consider using a PEP8 pre-commit hook, see http://docs.obspy.org/master/coding_style.html?highlight=hook